book/website/ethical-research/ethics-open-source-governance.md

```diff
@@ -0,0 +1,99 @@
+It's important that the governance model you select provides robust mechanisms for the level of community participation you are hoping to achieve with your project, with self-reflection being a useful starting point to guide your decision making. 
+
+(er-ethics-open-source-governance-continuous-process)=
+## Ethical Governance is a Continuous Process
+There are many ways in which your project can (and probably will) change over its lifetime, and some of these changes may be unexpected. 
+For example, your software might start being used in domains you have not considered, it might experience a faster growth rate than you have expected, or you may need to decide how to handle an influx of new contributors.
+
+This may also mean that you have to change the way that the project is run, to actively include new parts of the community, deal with a surge in contributions, or face new challenges.
+In this scenario, try and reflect the ways in which the governance of your project should evolve to reflect the ethics you commit to.
+For instance, is your governance structure built to allow the whole community to participate and influence a project, that will in turn influence themselves and their work?
+
+It may feel like a giant leap to go from, say, a free-form do-ocracy to something more structured, or to give up your position of benevolent dictator, or to introduce different voting rules for the steering committee, but if you want your project to reflect your ethics, you will need to allow your ethics to impact the project, and iterate on this process. 
+It is a bit like putting debugging checkpoints in the way you run your project: run, but halt to reflect if you encounter a state that is unethical, based on your definition of ethical. 
+
+In short: `if (ethical == true) { // Do stuff } else { // FIXME }`
+
+(er-ethics-open-source-governance-action)=
+## Ethics Need Governance to become Actionable
+Open source project communities may already have an implicit set of ethics regarding their project in place. 
+However, unless these ethics are being made explicit, and inform a structured model of governance, they will remain inconsequential or, as Jo Freeman has put it in her essay [“The Tyranny of Structurelessness”](https://www.google.com/url?q=https://www.jofreeman.com/joreen/tyranny.htm&sa=D&source=docs&ust=1652716021793901&usg=AOvVaw2m6_oS7n1u7IvfHamGJDBs), impotent.
+
+In order to empower your project to act on your ethics, it will need some governance model to adhere to. 
+There already exist a number of more-or-less established governance models, and you can take a look at the resources section of this chapter for links to more information on them. 
+Keep in mind, though, that not all of them may cater well for an ethical framework, or the specific ethical framework you have in mind, in fact….
```

Comment:
```suggestion
To empower your project to act on your ethics, it will need some governance model to adhere to. 
There already exist a number of more-or-less established governance models, and you can take a look at the resources section of this chapter for links to more information on them. 
Keep in mind, though, that not all of them may cater well for an ethical framework or the specific ethical framework you have in mind.
```

Reply:
was that an incomplete sentence? (ended with in fact ...)

Reply:
Ah that's an artefact of the blog post :)